### PR TITLE
Refacto/fasthttp on admin

### LINKED ISSUE #70: Implement FastHTTP on Admin API
**Is your feature request related to a problem? Please describe.**

Today, the reverse proxy is based on [fastHTTP](https://github.com/valyala/fasthttp), and the administration is based on [net/http](https://golang.org/pkg/net/http/). Two problems emerge from that : 

* Two different technologies does the same thing (http server) for the same codebase, which can be hard to maintain
* FastHTTP is more efficient and handle http servers problematic is a more appreciable way.

**Describe the solution you'd like**

Administration server should be based on fastHTTP

**Describe alternatives you've considered**

* Keep going like we do now. 

    The biggest problem is if we can't go further in the development of the project because we have to rely on fastHTTP. For now on, the project is small enough to consider refactor it without wasting too much time. If the administration application grow too much, it will be very painful to do so.

* Use fastHTTPAdaptador

    It could be an option, yet it seems that it implies keeping net/http AND fastHTTP which doesn't seem conceptually right, especially at the begining of the project where we can do a real transition.

**Additional context**

### PR DESCRIPTION
**Pull request by Zyigh**
# PR Details

Change net.http to fasthttp in administration API

## Description

Change net.http to fasthttp in administration API

## Related Issue

Closes #70 

## Motivation and Context

These changes helps to keep consistancy on code as the proxy is based on fasthttp

## How Has This Been Tested

This was test by hand on Mac OS 10.15.6  
It also works in the docker environment provided in the project

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: database/initialize.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func HandleFirstStart() error {
 		return nil
 	}
 	utils.Logger.Info("First start detected, generating default data...", nil)
-	utils.Logger.Info("Generating first admin user...", nil)
+	utils.Logger.Info("Generating first administration user...", nil)
```
Comment (qraimbault) [inline database/initialize.go]: Why did you make this one more explicit white at line 47 of administration/server.go you cutted it to "admin" ?

Comment (Zyigh) [reply database/initialize.go]: This file was auto edited by IDE. This should'nt be part of the PR, and if there is a debate about wording in this file, this should create an issue.

Changes on this files are removed from this PR

File: administration/controllers/service.go [hunk 2]
```diff
@@ -2,72 +2,75 @@ package controllers
...
 	var service models.Service
-	decoder := json.NewDecoder(r.Body)
-	if err := decoder.Decode(&service); err != nil {
-		w.WriteHeader(http.StatusBadRequest)
+	if err := json.Unmarshal(context.Request.Body(), &service); err != nil {
+		context.Response.SetStatusCode(http.StatusBadRequest)
 		return
 	}
+
+	message := fmt.Sprintf("There was an error Bro ! : %s", context.Request.Body())
```
Comment (qraimbault) [inline administration/controllers/service.go]: `Bro !` ?

Comment (qraimbault) [reply administration/controllers/service.go]: Also in this particular case there is no error since you handled the potential error above

**Review by qraimbault — CHANGES_REQUESTED** [review]

File: database/initialize.go (hunk 1 above)
Comment (qraimbault) [reply database/initialize.go]: Fine 👍

**Review by qraimbault — APPROVED** [review]

**Pull request merged**